node-data/src/lib.rs

```diff
@@ -110,3 +111,17 @@ where
     let hex = bs58::encode(t).into_string();
     serializer.serialize_str(&hex)
 }
+
+pub fn get_current_timestamp() -> u64 {
+    SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .map(|n| n.as_secs())
+        .expect("This is heavy.")
+}
+
+pub fn get_current_timestamp_safe() -> u64 {
+    SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .unwrap_or_default()
+        .as_secs()
+}
```

Comment:
There is no need to have this as `safe` since the above function will always be safe unless the OS timestamp is set before than 1970

Reply:
I did this because there were two cases where `unwrap_or_default` was used and it's not compatible with `expect`.
Should I just use the "unsafe" version everywhere then?

Reply:
Yes